<launch>
    <node pkg="nlink_parser" type="linktrack" name="linktrack">
        <param name="port_name" value="/dev/ttyUSB0"/>
        <param name="baud_rate" value="921600"/>
    </node>
    <node pkg="spacurs" type="usv.py" name="usv" output="screen"/>
    <node pkg="spacurs" type="xbee.py" name="xbee">
        <param name="port" value="/dev/ttyUSB1"/>
        <param name="baud_rate" value="115200"/>
        <param name="node_id" value="2"/>
    </node>
    <node pkg="spacurs" type="position.py" name="position"/>
    <node pkg="spacurs" type="visualize.py" name="visualize"/>
</launch>